mleap-core/src/main/scala/ml/combust/mleap/core/feature/MathUnaryModel.scala

```diff
@@ -56,3 +61,28 @@ case class MathUnaryModel(operation: UnaryOperation) extends Model {
   override def outputSchema: StructType = StructType(
     "output" -> ScalarType.Double.nonNullable).get
 }
+
+object LogitHelper { 
+  val CLIPPING_MIN = 0.000001
+  val CLIPPING_MAX = 0.999999
+  val LOGIT_MIN = 0.0
+  val LOGIT_MAX = 1.0
+  
+  def clipToLogitRange(value: Double) : Double = { 
+    if (value <= CLIPPING_MIN) { 
+      return CLIPPING_MIN
+    } 
+    if (value >= CLIPPING_MAX) { 
+      return CLIPPING_MAX
+    } 
+    return value
+  } 
+
+
+  def logit(value: Double): Double = {
+    val clippedValue = clipToLogitRange(value) 
+    val logitInstance = new Logit(LOGIT_MIN, LOGIT_MAX)
```

Comment:
I think `Logit()` is equivalent to `Logit(0.0, 1.0)`, so these constants should be unnecessary

https://commons.apache.org/proper/commons-math/javadocs/api-3.0/org/apache/commons/math3/analysis/function/Logit.html#Logit()

Reply:
Good point